create-a-discovery-register-environment.md

```diff
@@ -0,0 +1,83 @@
+# Instructions for creating discovery register environments
+
+In these instructions, the new register to be created is called `meat-cuts`
+
+## 1. AWS: edit the config file
+
+Tell everyone you're creating a new register, so that you don't both do it at
+the same time.
+
+Log into AWS https://openregister.signin.aws.amazon.com/console
+
+Go to the [S3 bucket](https://s3.console.aws.amazon.com/s3/buckets/openregister.discovery.config/?region=us-east-2&tab=overview)
+or browse to `Services > Storage > S3 > openregister.discovery.config`
+
+Download this
+[file](https://s3.console.aws.amazon.com/s3/object/openregister.discovery.config/multi/openregister/paas-config.yaml?region=us-east-2&tab=overview)
```

Comment:
might be easier to do this with the AWS CLI instead? https://docs.aws.amazon.com/cli/latest/reference/s3/index.html we can then wrap these in shell scripts like `download_config.sh` or something?